ui/src/app/views/workspace/component.ts

```diff
@@ -0,0 +1,48 @@
+import {Component, OnInit} from '@angular/core';
+import {Router} from '@angular/router';
+
+import {CohortsService} from 'generated';
+import {Repository} from 'app/models/repository';
+import {RepositoryService} from 'app/services/repository.service';
+import {User} from 'app/models/user';
+import {UserService} from 'app/services/user.service';
+
+
+const cohortList = [];
+
+
+@Component({
+  styleUrls: ['./component.css'],
+  templateUrl: './component.html',
+})
+export class WorkspaceComponent implements OnInit {
+  repositories: Repository[] = [];
+  user: User;
+  // TODO: Pull cohortList from external source
+  cohortList = cohortList;
+  constructor(
+      private router: Router,
+      private userService: UserService,
+      private repositoryService: RepositoryService,
+      private cohortsService: CohortsService
+  ) {}
+
+  ngOnInit(): void {
+    this.userService.getLoggedInUser()
+        .then(user => this.user = user);
+    this.cohortsService.getCohortsInWorkspace('123', '123').subscribe(
+          cohortsReceived => {
+            for (const cohortIn of cohortsReceived){
+              cohortList.push(cohortIn);
```

Comment:
Are you calling `push` in an array because reassigning the whole list (`this.cohortList = cohortsReceived`) doesn't work?